src/Avalonia.Visuals/Thickness.cs

```diff
@@ -234,16 +250,12 @@ public static Thickness Parse(string s)
         /// </returns>
         public override bool Equals(object obj)
         {
-            if (obj is Thickness)
+            if (!(obj is Thickness))
```

Comment:
Suggestion: a really simple way to implement `Object.Equals` is:

```cs
public override bool Equals(object obj) => obj is Thickness other && Equals(other);
```

Reply:
And done :)